lib/news_scraper/transformers/article.rb

```diff
@@ -20,22 +19,26 @@ def transform
         transformed_response.merge(root_domain: root_domain)
       end
 
+      def self.scrape_patterns
```

Comment:
I would rather not cause this to reload the yml file every instantiation of `Transformers::Article` for the sake of the `Trainer`.

Keep the constant and add a "self.reload_scrape_patterns" that reloads the constant?